ui/src/components/card/card.styles.ts

```diff
@@ -0,0 +1,35 @@
+import { dripStitches } from '@/theme';
+
+const { styled } = dripStitches;
+
+export abstract class CardStyles {
+  static readonly Container = styled('div', {
+    backgroundColor: '$slate2',
+    borderRadius: '$xlh',
+    padding: '$7',
+    borderStyle: 'solid',
+    borderColor: '$slate6',
+    borderWidth: '$default',
+  });
+
+  static readonly Heading = styled('h3', {
+    color: '$slate12',
+    fontSize: '$xl',
+    fontWeight: '$medium',
+  });
+
+  static readonly Body = styled('div', {
+    pt: '$5',
+  });
+
+  static readonly CardText = styled('div', {

```

Comment:
Seems like we don't need `Card`here.
```suggestion
  static readonly Text = styled('div', {
```